charts/osm/templates/cleanup-hook.yaml

```diff
@@ -26,3 +26,4 @@ spec:
             - -c
             - >
              kubectl delete --ignore-not-found meshconfig -n '{{ include "osm.namespace" . }}' osm-mesh-config;
+             kubectl delete --ignore-not-found meshconfig -n '{{ include "osm.namespace" . }}' preset-mesh-config;
```

Comment:
Why is this required? Shouldn't this be automatically deleted by Helm?

Reply:
the configmap also previously was installed by helm but this cleanup hook handled it, hence I added the preset-mesh-config

cc @nojnhuh

Reply:
We added the configmap deletion here in anticipation of removing it from the chart. Since the preset-mesh-config is part of the chart, we don't need to delete it here.

Reply:
Hmm, unsure why this would be necessary. @nojnhuh was there a reason to explicitly remove this?

Reply:
thanks for the clarification @nojnhuh , I have removed the preset-mesh-config